src/bin/pg_dump/pg_dump.h

```diff
@@ -706,23 +706,6 @@ typedef struct _SubscriptionInfo
 } SubscriptionInfo;
 
 /*
-<<<<<<< HEAD
-=======
- * We build an array of these with an entry for each object that is an
- * extension member according to pg_depend.
- */
-typedef struct _extensionMemberId
-{
-	CatalogId	catId;			/* tableoid+oid of some member object */
-	ExtensionInfo *ext;			/* owning extension */
-} ExtensionMemberId;
```

Comment:
Should we remove the `ExtensionMemberId` type from `src/tools/pgindent/typedefs.list`?